api/naming/naming.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/docker/swarmkit/api"
 )
 
+const (
+	// PredefinedLabel identifies internally allocated swarm resources
+	PredefinedLabel = "com.docker.swarm.predefined"
```

Comment:
Same comment - we have a `com.docker.swarm.internal` label already around. Is this any different? Could we just re-use the same one?